src/pages/dormitory/call_names.tsx

```diff
@@ -61,39 +126,29 @@ export default function CallNamesPage() {
               <div>
                 {(() => {
                   if (characterKey === _characterKey) {
-                    return callNameInfo.me.map(part => (
-                      <p style={outlineStyle} key={part} className="me">
-                        {part}
-                      </p>
-                    ))
+                    return callNameInfo.me.map(part =>
+                      getCell(characterKey, part, "me")
+                    )
                   }
 
                   if (callName == undefined) {
                     return (
-                      <p style={outlineStyle} className="unknown">
+                      <p className="unknown" style={outlineStyle}>
                         ?
```

Comment:
ちょっとこのプルリクエストとは違いますが気づいたので。。
基本的に日本語のサイトはフォントの都合とかもあって全角記号の方が良いのかなと思いました！
```suggestion
                        ？
```

Reply:
なるほど.　Suggestion ありがとうございます！  
そういえば, ここらへんのルールがあったりするかなと思いましたが, エディターのほうの docs に記載がありましたね.  
見落としてました.　以後それに従うようにします……！  

> - アルファベットは半角
> - 括弧は全角
>   - ただし、中身が全て半角文字の場合に限り、括弧も半角
> - です/ます調
> - 項目名は名詞で終わる（体言止め）
> - button の中身は「～する」  
>
> ―― [voicevox/docs/UX・UIデザインの方針.md](https://github.com/VOICEVOX/voicevox/blob/main/docs/UX%E3%83%BBUI%E3%83%87%E3%82%B6%E3%82%A4%E3%83%B3%E3%81%AE%E6%96%B9%E9%87%9D.md#%E6%96%87%E4%BD%93) より